lib/new_relic/agent/instrumentation/action_mailbox.rb

```diff
@@ -0,0 +1,27 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require 'new_relic/agent/instrumentation/action_mailbox_subscriber'
+
+DependencyDetection.defer do
+  named :action_mailbox
+
+  depends_on do
+    defined?(ActiveSupport) &&
+      defined?(ActionMailbox) &&
+      ActionMailbox.respond_to?(:gem_version) && # 'require "actionmalbox"' doesn't require version...
```

Comment:
The comment has a misspelling of 'actionmailbox'!

Reply:
Rats! There's also supposed to be an underscore in there, "action_mailbox"! Fixed with 0aa67aed20990eba253ae8dd1dd3661a2406929d